elastic/kibana
# [Cloud Security] UI counter telemetry

## Summary
In this PR, I added a UI counter to the Cloud Security component.

A new utility, `ui_metrics`, was introduced to the `kbn-cloud-security-posture-common` package. This allows it to be utilized by both the Security Solution plugin and the Cloud Security plugin. The purpose of integrating it into the Security Solution plugin is to track the Cloud Security component within the Security plugin.

## How to test it
Interact with one of the tracked components and call the telemetry API using the dev tools:
```
POST kbn:/internal/telemetry/clusters/_stats?apiVersion=2
{
  "unencrypted": true
}
```
Look for `"appName": "cloud-security"` under the `ui_counters` section and confirm that the relevant component tracker has increased.
